independent-projects/qute/core/src/main/java/io/quarkus/qute/Variant.java

```diff
@@ -1,12 +1,17 @@
-package io.quarkus.qute.api;
+package io.quarkus.qute;
 
 import java.util.Locale;
 import java.util.Objects;
 
 /**
  * Media type, locale and encoding.
  */
-public class Variant {
+public final class Variant {
+
+    public final static String TEXT_HTML = "text/html";
+    public final static String TEXT_PLAIN = "text/plain";
+    public final static String TEXT_XML = "text/xml";
```

Comment:
Note that XML can be `application/xml` in many cases, but I don't think it matters here. https://www.ietf.org/rfc/rfc2376.txt

Reply:
Yes, I know. But in our case we guess the content type from a file suffix and set always `text/xml` for `*.xml` templates.